.circleci/config.yml

```diff
@@ -252,7 +282,7 @@ workflows:
           requires:
             - build_x86_64
             - build_aarch64
-      - integration_test:
+      - calico_test:
```

Comment:
We probably still want the regular `integration_test` here.

Reply:
Yeah this is just test on my local account, should have probably marked it as draft :D..will revert it soon.